src/cms/templates/pages/page_tree_node.html

```diff
@@ -33,6 +33,13 @@
             {{ page_translation.get_status_display }}
         </a>
     </td>
+    <td>
+        <a href="{% url 'edit_page' page_id=page.id region_slug=region.slug language_slug=LANGUAGE_CODE %}" class="block py-3 px-2 text-gray-800">
+            {% for tag in page.backend_translation.tags %}
```

Comment:
```suggestion
            {% for tag in page_translation.tags %}
```
`backend_translation` is the page translated into the current **UI language** of the user, but all the information in the page tree should correspond to the current selected **content language**, so I'd suggest to show the tags of `page_translation` here. So if the user has set his UI language to German but looks at the page tree of all English translations, the "Empty" label should be shown if the English translation is empty, not the German translation.
Switching content languages is very unintuitive at the moment, you'd have to click on the tiny language flags to change it.